src/components/Layout/Layout.jsx

```diff
@@ -20,6 +21,7 @@ const Layout = ({ children, seo, slug }) => {
       <Navbar slug={slug} />
       <SEO title={siteTitle} description={siteDescription} />
       {children}
+      <Stories />
```

Comment:
don't add it here

create and put it in

`src\templates\story-list-page\index.jsx`

Reply:
inside `src\templates\story-list-page\index.jsx`

add this
```
import React from 'react'
import Stories from '../../components/Stories';
import Layout from '../../components/Layout';

const StoryPage = () => {
const seo = {
siteTitle: 'Hello'.
siteDescription: 'World'
}
return (
<Layout path="/stories" seo={seo}><Stories /><Layout />
)
}

Reply:
refer to the other template pages for reference